CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ title: Changelog
 
 This is a list of changes/improvements that were introduced in ProxyManager
 
+## 2.0.1
+
+Total issues resolved: **2**
```

Comment:
See the `## 2.0.0` section for an example. These two are under section `### Added` and `### Fixed`